### ISSUE TITLE
Hub Error: Index `products_en` not found. 

### ISSUE THREAD

**Issue by poppabear8883** [getcandy/getcandy#13]
Getting this error after clicking on `Products` via the Hub.

```
MeiliSearch\Exceptions\ApiException
Index `products_en` not found. (View: /home/vagrant/projects/karve/vendor/getcandy/getcandy/resources/views/livewire/pages/products/index.blade.php)
```

**Comment by poppabear8883** [comment]
Executing `php artisan scout:index products_en` fixes this issue. Why are the Indexes not being created ? Is this related to Issue #8 ??

**Comment by glennjacobs** [comment]
Can you confirm what version of the package you're running?

We did have an issue with index creation, but I believe it was resolved in the latest version.

**Comment by poppabear8883** [comment]
@glennjacobs as seen in issue #8 I commented showing the update to `2.0-beta4` and still receiving the same issue. I executed the command `php artisan scout:index getcandy_orders && php artisan getcandy:meilisearch:setup` manually to bypass that error. This completed succesfully, but the above issue remained. I than ran `php artisan scout:index products_en` and this got rid of this specific issue making me believe that there is still an issue with the indexes in some manner ?
